src/resources/js/components/CoursesCarousel.js

```diff
@@ -0,0 +1,367 @@
+Vue.component('courses-carousel', {
+    template: 
+    `
+    <div>
+    <h2 class="pl-4 font-weight-bold" style="margin-top: 2em">Let's start learning, MusahMusah</h2>
+    <div v-for="(item, index) in items">
+        <h3 class="pl-4 p-3 font-weight-bold" :title="item.title">Top courses in <a href="" style="color: blue">{{ item.category }} </a></h3>
+        <div class="row visible-courses">
+            <div class="col-12">
+            <div :id="'myCarousel'+index" class="carousel slide" data-ride="carousel" data-interval="0">
```

Comment:
kindly convert this to a re usable component, and lets try and not be hardcoing data, you can always spool a dummy array for this and loop through